tests/continental_extension_mixed_compositions_particles.prm

```diff
@@ -0,0 +1,378 @@
+# This is a simplified version of the continental extension cookbook to test
+# the combination of different compositional field discretizations and
+# particles
+
+####  Global parameters
+set Dimension                              = 2
+set Start time                             = 0
+set End time                               = 10000
```

Comment:
```suggestion
set End time                               = 30000
```

Reply:
We've had issues in the past with this and similar cookbooks where an issue was not apparent until at least the 2nd time step.